tests/test_layers_stack.py

```diff
@@ -1,16 +1,84 @@
 import unittest
 
-# GeonotebookStack
-## Instantiate GeonotebookStack with list of non GeonotebookLayers should throw Exception
-
-## Instantiate GeonotebookStack with list of GeonotebookLayers (e.g. 3 layers 'foo', 'bar', 'baz')
-### GeonotebookStack.find('foo') should return GeonotebookLayer
-### GeonotebookStack.find('derp') should return None
-### GeonotebookStack.find(lambda g: g.name == 'foo') should return GeonotebookLayer
-### GeonotebookStack.find(lambda g: g.name == 'derp') should return None
-### GeonoteobokStack.indexOf('foo') should return 0
-### GeonoteobokStack.indexOf('bar') should return 1
-### GeonoteobokStack.indexOf('baz') should return 2
-### GeonoteobokStack.indexOf('derp') should return None
-### GeonotebookStack[0] should return GeonotebookLayer and name should equal 'foo'
-### GeonotebookStack['foo'] should return GeonotebookLayer and name should equal 'foo'
+from geonotebook import layers
+
+
+class TestGeonotebookStack(unittest.TestCase):
+
+    def setUp(self):
+
+        self.stack = self.generate_stack()
+
+    def test_geonotebookstack_instantiated_with_geonotebooklayers(self):
+        """ Instantiate GeonotebookStack with list of
+        non GeonotebookLayers should throw Exception """
+
+        wrong_layers = [1, 2, 3]
+
+        self.assertRaises(AssertionError, layers.GeonotebookStack,
+                          layers=wrong_layers)
+
+    @staticmethod
+    def generate_stack():
+        """ Generates 3 geonotebook instances """
```

Comment:
@kotfic https://github.com/OpenGeoscience/geonotebook/pull/10/commits/9f888a28de07fec35c1d0147243586f04beb004d fixes it.